Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM openjdk:8-jre-slim
 
 VOLUME /tmp
-ARG JAR_FILE=actionexportersvc-*.jar
+ARG JAR_FILE=census-rm-actionexportersvc-*.jar
 RUN apt-get update
 RUN apt-get -yq install curl
 RUN apt-get -yq clean
-COPY target/$JAR_FILE /opt/actionexportersvc.jar
```

Comment:
This step fails for me because there is more than one Jar file that matches the JAR_FILE pattern.

Reply:
Are you able to reproduce this, the pattern above seems to be repeated in other services, such as the case-svc?